sdk/internal/state_locker.go

```diff
@@ -1,110 +0,0 @@
-// Copyright The OpenTelemetry Authors
```

Comment:
I think something went wrong when merging to my own repo. I'll have to check why these files were deleted...